notebooks/117-optimize-preprocessing/117-optimize-preprocessing.ipynb

```diff
@@ -0,0 +1,498 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    resized_image = np.array([104, 117, 123]).reshape((3, 1, 1))
</pre><p>it is not resized image, it is mean_values. Probably order should be slightly different (resized_image - mean_values, then create input_tensor, it helps to avoid additional reshaping) and missed step for converting input data type from uint8 to float32</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/811/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
yes, thank you, changed